plugin/metrics/README.md

```diff
@@ -19,8 +19,9 @@ The following metrics are exported:
 * `coredns_dns_response_size_bytes{server, zone, proto}` - response size in bytes.
 * `coredns_dns_responses_total{server, zone, rcode, plugin}` - response per zone, rcode and plugin.
 * `coredns_plugin_enabled{server, zone, name}` - indicates whether a plugin is enabled on per server and zone basis.
+* `coredns_https_responses_total{server, scode}` - responses per server and http status code.
```

Comment:
I'm also wondering about this. Since this is specific to DNS responses over https, perhaps we should namespace it that way.

```suggestion
* `coredns_dns_https_responses_total{server, scode}` - responses per server and http status code.
```

Reply:
Ok, done